_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -23,6 +23,8 @@ ng add @spartacus/schematics --ssr
 
 You have now added SSR support to your Spartacus application. No further steps are required.
 
+If you are experiencing issues, you can refer to the SSR Troubleshooting section at the bottom of the <a href="{{ site.baseurl }}{% link _pages/dev/ssr/server-side-rendering-optimization.md %}">Server-Side Rendering Optimization</a> page.
+
```

Comment:
@gladius-mtl Is there a way to create a link to an anchor inside a page?
I assumed no since I did not find any examples in the docs.
I would have like to provide a direct link to the troubleshooting section: https://sap.github.io/spartacus-docs/server-side-rendering-optimization/#troubleshooting